### PR TITLE
Aaronsharris/create layout

### PR DESCRIPTION
**Pull request by aaronsHarris**
I touched all these: 
Layout.jsx
Layout.scss
Nav.jsx
Nav.scss
MobileMenu.jsx
MobileMenu.scss
Index.css
App.css
App.js

### CODE REVIEW

File: client/src/layout/Layout.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import MobileMenu from "./MobileMenu/MobileMenu";
+import Nav from "./Nav/Nav";
+import './Layout.scss'
+
+const Layout = (props) => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const toggle = () => {
+    setIsOpen(!isOpen);
+    console.log('clicked');
```
Comment (yeezick) [inline client/src/layout/Layout.jsx]: now that we know `toggle` works, we can remove this console.log before merging into development

File: client/src/layout/MobileMenu/MobileMenu.scss [hunk 2]
```diff
@@ -0,0 +1,34 @@
+.mobile-menu-container.active{
+  display: grid;
+  align-items: center;
+  width: 100%;
+  height: 90%;
+  background-color: darkgray;
+  transition: 0.3s ease-in-out;
+}
+.mobile-menu-container{
```
Comment (yeezick) [inline client/src/layout/MobileMenu/MobileMenu.scss:9]: now that we are leveraging `scss`, let's make the most of it by consolidating these classes with something like 
```js
.mobile-menu-container {
display: none;

::active {
 display: grid;
  align-items: center;
  width: 100%;
  height: 90%;
  background-color: darkgray;
  transition: 0.3s ease-in-out;
  }
}
```

File: client/src/layout/MobileMenu/MobileMenu.scss [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  color: #000;
+  font-size: 1.5rem;
+  text-decoration: none;
+  list-style: none;
+  cursor: pointer;
+  transition: 0.2s ease-in-out;
+}
+.nav-link:hover {
+  color: blueviolet;
+}
```
Comment (yeezick) [inline client/src/layout/MobileMenu/MobileMenu.scss:34]: let's do the same here with 
```js
.nav-link {
....
::hover {
color: blueviolet;
}
}

File: client/src/layout/Nav/Nav.scss [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  padding-right: 10px;
+  cursor: pointer;
+}
+i {
+  background-color: black;
+  width: 32px;
+  height: 4px;
+  margin: 4px;
+  border-radius: 2px;
+}
```
Comment (yeezick) [inline client/src/layout/Nav/Nav.scss:26]: same here with the i element
```js
.menu-btn {
....previous properties
i {
...properties of i
}
}
```

File: client/src/layout/Layout.jsx (hunk 1 above)
Comment (aaronsHarris) [reply client/src/layout/Layout.jsx]: 👍

File: client/src/layout/MobileMenu/MobileMenu.scss (hunk 3 above)
Comment (aaronsHarris) [reply client/src/layout/MobileMenu/MobileMenu.scss:34]: 👍

**Review by yeezick — APPROVED** [review]

**Pull request merged**